src/shared/modules/dbMeta/dbMetaDuck.js

```diff
@@ -104,6 +104,7 @@ export const allowOutgoingConnections = state =>
 export const credentialsTimeout = state =>
   getAvailableSettings(state)['browser.credential_timeout'] || 0
 export const getRemoteContentHostnameWhitelist = state =>
+  state[NAME]['browser.remote_content_hostname_allowlist'] ||
```

Comment:
Done this way because getAvailableSettings seems to be using initial state when not found already.